core/src/main/java/org/infinispan/statetransfer/OutdatedTopologyException.java

```diff
@@ -5,19 +5,44 @@
 /**
  * An exception signalling that a command should be retried because it was executed with an outdated
  * topology.
- *
+ * <p>
  * This can happen for non-tx caches, if the primary owner doesn't respond (either because it left the
  * cluster or because this particular cache is no longer running).
  *
  * @author Dan Berindei
  * @since 6.0
  */
 public class OutdatedTopologyException extends CacheException {
+
+   //TODO? should we create a special externalizer to return the cached exception? Is it ever worth to send the stack trace/message when originated remotely?
```

Comment:
Since the ST isn't helpful, we shouldn't record it at all (the Exception(msg, cause, false, false) ctor)